connectors/sources/tests/fixtures/mysql/Makefile

```diff
@@ -1,31 +0,0 @@
-.PHONY: run-stack stop-stack load-data
```

Comment:
Why was this file removed? It's not needed any more?

Reply:
it adds a lot of friction and requires to allow overwriting
- custom python
- custom venv

I am calling everything from `ftest` now and planning to do this for all fixtures.

so the "standard" would become a few python functions

Reply:
Cool, makes sense!